app/helpers.php

```diff
@@ -73,6 +73,17 @@ function format_date_form_field($model, $field, $defaut = 'MM/DD/YYYY')
 }
 
 /**
+ * Formate a timestamp using Carbon for display in a view.
+ *
+ * @param  string  $timestamp
+ * @return /Carbon/Carbon
+ */
+function format_timestamp_for_display($timestamp)
```

Comment:
It's not a Carbon instance when retrieved. It's the Reportback data coming from Phoenix, which is just a straight-up regular object.